ferrocene/doc/user-manual/src/overview.rst

```diff
@@ -5,8 +5,12 @@ Overview
 ========
 
 This Manual describes the use of Ferrocene |ferrocene_version|, the |iso_ref|
-and |iec_ref| qualified version of the Rust toolchain. Ferrocene is based on rustc
-version |rust_version|.
+and |iec_ref| qualified version of the Rust toolchain.
+
+Ferrocene is based on:
+
+* rustc version |rust_version|.
+* rustfmt version |rustfmt_version|.
```

Comment:
Should or must we mention the rustfmt version here?

Reply:
Yes.

Reply:
Okies. Then I am gonna add the dynamic substitution.

Reply:
Dynamic substitution will be implemented in a follow-up PR